packages/@okta/vuepress-site/docs/reference/api/myaccount/index.md

```diff
@@ -297,6 +297,87 @@ curl -XPUT 'https://${yourOktaDomain}/api/v1/myaccount/directoryProfile' -H 'Aut
 }
 ```
 
+### Send Phone Challenge
+
+<ApiOperation method="post" url="/idp/myaccount/phones/{id}/challenge"/>
+
+Send a phone challenge via one of the two methods: `SMS` or `CALL`. This API handles resend challenge (retry) as well.
+
+Upon a successful challenge, the user will receive a verification code via either `SMS` or `CALL`. The verification code can be used to verify the phone number by calling `POST /idp/myaccount/phones/{id}/verify` endpoint. The verification code would expire in 5 minutes.
+
+This endpoint is supposed to be called no more than once every 30 seconds.
+
+#### Request path parameters
+
+`id`: the id of the phone factor, which can be found in the response when a new phone is created successfully (`POST /idp/myaccount/phones`) or phone(s) is retrieved (`GET /idp/myaccount/phones`).
```

Comment:
We should put these parameters in table format. Is `id` a string type? Maybe instead of including the calls in the description, we could link to them in the docs?
```suggestion
| Parameter  | Type   | Description                                       |
| ---------- | ------ | ------------------------------------------------- |
| `id` | String | The id of the phone factor. Found in the response when a new phone is created successfully (`POST /idp/myaccount/phones`) or phone(s) is retrieved (`GET /idp/myaccount/phones`)  |
```

Reply:
Good point, will change it to a table. `id` is of string type. All paths variables and request properties in /challenge and /verify are either string or boolean.